ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2185,6 +2185,84 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetalds-ipi-ovn-ipv6-fips-f4
+  cron: 34 18 3,7,11,15,20,25,29 * *
```

Comment:
I am not sure if `equinix` quota can meet the high test frequency, cc @pamoedom and @aleskandro to review.

Reply:
if equinix cannot be handled that.  we can reduce basic job, eg. 
 ipv4 job to f14 and keep dualstack to f7 
and ipv6 to f14 and dualstack-v6v4to f7.

so that mean each version dualstack can be tested for Z stream release one week IMO

Reply:
How do we get more quota?  We need to communicate all the teams that adding new IP Families increases the testing matrix.

Reply:
We should test local gateway and IPsec with dualstack as well.

Reply:
Hi, I'd use a lower frequency, although quota for equinix and for these jobs in particular should not be an issue as we spawn one only machine.

Also consider that these ipv6 workflows in the baremetalds have had a very low pass ratio in the time due to automation issues in dev-scripts. Perhaps, lowering the frequency and migrating these jobs to rdu2 once the new infra will be ready is a good option.

Reply:
Thanks. 
updated the test frequency to
ipv4 job to f14 and keep dualstack to f7
and ipv6 to f14 and dualstack-v6v4to f7.

and seems ipv6 looks good for me using dev-scripts for most of times.  I did not see any failed yet until now. 
However if rdu2 is better in future, we can move it.